### PR TITLE
Fixes "Maven Archetypes" (initial version)

### PR DESCRIPTION
**Pull request by yasserzamani**
Thanks a lot Regular Expressions 😗

### CODE REVIEW

File: source/maven-archetypes/index.md [hunk 1]
```diff
@@ -3,155 +3,146 @@ layout: default
...
-#Struts 2 Maven Archetypes# {#PAGE_108820}
+# Struts 2 Maven Archetypes
 
-Struts 2 provides several [Maven](http://maven\.apache\.org/)^[http://maven\.apache\.org/] archetypes that create a starting point for our own applications\.
+Struts 2 provides several [Maven](http://maven.apache.org/) archetypes that create a starting point for our own applications.
 
 **Contents**
+{:.no_toc}
 
-###Quickstart###
```
Comment (lukaszlenart) [inline source/maven-archetypes/index.md]: It would be better to use `##` instead of `###`

File: source/maven-archetypes/index.md [hunk 2]
```diff
@@ -3,155 +3,146 @@ layout: default
...
+* Will be replaced with the ToC, excluding a header
+{:toc}
 
-A recommended way to start with Struts2 archetypes is to work with the archetype catalog\.
+### Quickstart
```
Comment (lukaszlenart) [inline source/maven-archetypes/index.md]: It would be better to use `##` instead of `###`

Comment (yasserzamani) [reply source/maven-archetypes/index.md]: Oh... Thank you very much for that 👍 . I also fixed others :)

**Comment by lukaszlenart** [comment]
Nice, 👍

**Pull request merged**